Libraries/WebSocket/RCTReconnectingWebSocket.m

```diff
@@ -11,7 +11,7 @@
 
 #import <React/RCTConvert.h>
 #import <React/RCTDefines.h>
-#import <fishhook/fishhook.h>
+#import "fishhook.h"
```

Comment:
I don't think this change is correct. The `fishhook.h` file is not in the same directory, so importing it needs to be done through the fishhook library, e.g. `#import <fishhook/fishhook.h>`.

It might be the case that it's actually included under `React`, instead (see https://github.com/facebook/react-native/pull/16192#issuecomment-337428712).